README.md

```diff
@@ -10,6 +10,13 @@ If you have a working Go installation, getting the binary should be as simple as
 go get github.com/xperimental/nextcloud-exporter
 ```
 
+or with Docker:
```

Comment:
I agree that the readme is a bit outdated by now and needs an update, but I would leave it out of this PR except for the updates describing the current change.

Reply:
Alright.